requirements.txt

```diff
@@ -22,3 +22,4 @@ spacy
 pyyaml
 textflint
 ujson>=4.2.0
+urllib3[secure]
```

Comment:
This is probably somehow necessary, but I was wondering what we added in the PR that made it dependent on urllib3 (if you know)?